fastlane/Fastfile

```diff
@@ -659,24 +670,54 @@ REPOSITORY_NAME="WordPress-Android"
       play_store_app_title: {desc:"title.txt", max_size: 50}
     }
 
-    delete_old_changelogs(build: options[:build_number])
-    download_path = File.join(Dir.pwd, paths[:metadata_dir], 'android')
+    delete_old_changelogs(app: 'wordpress', build: options[:build_number])
+    download_path = File.join(Dir.pwd, app_values[:metadata_dir], 'android')
+    # The case for the source locale (en-US) is pulled in a hacky way, by having an {en-gb => en-US} mapping as part of the RELEASE_NOTES_LOCALES,
+    # which is then treated in a special way by gp_downloadmetadata by specifying a `source_locale: 'en-US'` to process it differently from the rest.
```

Comment:
This is a hack that took me a while to understand/undercover, as I first I didn't understand from the API (and the action's description and parameters doc which are a bit succinct) how we handled the case of the source locale and why we were using `en-gb` as a useless locale (we never use the translations for this one, only reference it to extract the source copy…).

Side note: I'm hoping to improve that logic at some point in the future, to make this not needed to add a "twisted" entry in the list of locales (with en-gb=>en-US mapping that stands out without any doc/explanation) and even not requiring us to have en-gb in GP anymore, as we can totally rely on the originals from any other locale already downloaded, for the same purpose and outcome.